1_Network_Structure.ipynb

```diff
@@ -659,10 +659,10 @@
     "<div class=\"alert-info\">\n",
     "\n",
     "#### Short Answer 3\n",
-    "Write a few sentences about the distance (number of edges) between nodes in this network, considering the following. \n",
+    "Write a few sentences about the distance (number of edges of the shortest path) between nodes in the _A Storm of Swords_ network `G_asoiaf`, considering the following. \n",
     "    \n",
-    "1. How long would you guess the typical shortest path between two nodes is? \n",
-    "2. How long would you guess the shortest path is between the two most distance nodes? \n",
+    "1. How long would you guess the typical distance between two nodes is? \n",
+    "2. How long would you guess the distance is between the two most distance nodes? \n",
     "    \n",
```

Comment:
I suggest not to change these two places from "shortest path" to "distance".  
They intent to highlight the definition of the "distance" by the length of the "shortest path" not any other paths.